app/controllers/users_controller.rb

```diff
@@ -0,0 +1,16 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.create(user_params)
+    redirect_to users_url
+  end
+
+  private
+
+  def user_params
+    params.require(:user).permit(:username, :password)
```

Comment:
Hey, are we logging in with just username and password? Or with email? Because the Login page (app/views/devise/sessions/new.html.erb line 7) has email as a requirement to log in.. Unless i am missing something.